samples/dashboards/demo/project-management/demo.css

```diff
@@ -1,6 +1,6 @@
 @import url("https://code.highcharts.com/css/highcharts.css");
 @import url("https://code.highcharts.com/css/datagrid.css");
```

Comment:
Should probably do this one as well ( --> https://code.highcharts.com/datagrid/css/datagrid.css). https://code.highcharts.com/css/datagrid.css is still there, but seems to be an older version

Reply:
I do not remember that we use  https://code.highcharts.com/datagrid/css/datagrid.css anywhere, should we?

Reply:
Looks like https://code.highcharts.com/dashboards/css/datagrid.css is also still an option, but maybe /datagrid would be better for future-proofing? cc @oysteinmoseng

Reply:
Good point but actually we don't use the DataGrid in this demo so it can be removed 😉

Reply:
😆